core/interop/godll/cs/README.md

```diff
@@ -0,0 +1,40 @@
+---
+languages:
+- csharp
+- go
+products:
+- dotnet-core
+page_type: sample
+name: "Calling Go functions from C# code"
+urlFragment: "golang-shared-c-dotnet-cs"
+description: "A simple example of using golang's cgo library and .NET's P/Invoke to call Go functions from C# code like a C-style API."
+---
+
+# Calling Go functions from C# code
+
+Go has a library called [cgo](https://golang.org/cmd/cgo/) that enables the creation of Go packages that call C code.  One of cgo's coolest features is that it enables Go packages to be compiled into C shared libraries, exposing Go functions as a C-style APIs.  This sample shows how we can compile a Go project into a Shared C **.dll** and call Go functions from within a .NET Core project using .NET's [P/Invoke](https://docs.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke).
+
+This project consists of 2 primary components:
+
+1. The .NET Core app that imports and calls functions from the Shared C libraries.  This code is found in `<SolutionDir>/SharedC/Program.cs`.
```

Comment:
I would remove `<SolutionDir>` and just include the relative path from this readme file. Perhaps you can update the other paths too?

```suggestion
1. The .NET Core app that imports and calls functions from the Shared C libraries.  This code is found in `./SharedC/Program.cs`.
```

Reply:
I will fix these.